elasticsearch/Chart.yaml

```diff
@@ -4,8 +4,8 @@ maintainers:
 - email: helm-charts@elastic.co
   name: Elastic
 name: elasticsearch
-version: 6.6.2-alpha1
```

Comment:
Can you remove this version bump please. All versions of the charts are bumped and released at the same time (just like with all of the elastic stack). I went into more details in this PR if you are interested in the reasons why: https://github.com/elastic/helm-charts/pull/40#discussion_r249422024

Reply:
reverted chart version bump